### ISSUE TITLE
Lists (accessibility)

### ISSUE THREAD

**Issue by lascellesabercrombie** [fac25/week4-alex-abdullah-joe-lisa#40]
This nesting was throwing up issues when I ran lighthouse testing:

https://github.com/fac25/week4-alex-abdullah-joe-lisa/blob/747215796dc20d9d5cd1be83c76599e2d39e5d70/src/templates.js#L94

`<li>` should be the direct child of `<ul>` - I think this makes your page more easily interpretable for users with screen readers. Perhaps apply the class to the `<li>` or put the `<div>` inside if necessary.
